scripts/mkcloud

```diff
@@ -1401,6 +1421,9 @@ function expand_steps
                     plain_with_upgrade_test)
                         runcmds="$runcmds `expand_steps plain` testpreupgrade addupdaterepo runupdate `expand_steps _upgrade` testpostupgrade"
                     ;;
+                    plain_with_ironic)
```

Comment:
Do we need another alias?
This can be called like this: `mkcloud plain ironic` (which is even shorter :) )